hlint/hlint.dhall

```diff
@@ -76,6 +76,11 @@ in [ rule.Arguments { arguments =
    , hintNote "fmap or (mapM f s)" "anyM f s" "Applying this hint would mean that some actions that were being executed previously would no longer be executed."
    , hintNote "or <$> mapM f s" "anyM f s" "Applying this hint would mean that some actions that were being executed previously would no longer be executed."
 
+   , warnSimple "getAlt (foldMap (Alt . f)) xs" "asumMap xs"
```

Comment:
Oops, my comment was badly formatted and this probably lead to possible error. The parenthesis here are wrong, it should be `"getAlt (foldMap (Alt . f) xs)"`.